plutus-use-cases/plutus-use-cases.cabal

```diff
@@ -1,242 +1,263 @@
-    ghc-options: -Wall -Wnoncanonical-monad-instances -Wunused-packages
-                 -Wincomplete-uni-patterns -Wincomplete-record-updates
-                 -Wredundant-constraints -Widentities
-                 -- See Plutus Tx readme
-                 -fobject-code -fno-ignore-interface-pragmas -fno-omit-interface-pragmas
-    build-depends:
-        base -any,
-        aeson -any,
-        bytestring -any,
-        containers -any,
-        plutus-core -any,
-        plutus-tx -any,
-        plutus-tx-plugin -any,
-        plutus-contract -any,
-        playground-common -any,
-        plutus-ledger -any,
-        plutus-ledger-api -any,
-        plutus-ledger-constraints -any,
-        plutus-script-utils -any,
-        template-haskell -any,
-        lens -any,
-        text -any,
-        prettyprinter -any,
-        hashable -any,
-        freer-simple -any,
-        streaming -any,
-        semigroups -any,
-        openapi3 -any,
-        freer-simple -any
-
-    if flag(defer-plugin-errors)
-        ghc-options: -fplugin-opt PlutusTx.Plugin:defer-errors
+  exposed-modules:
+    Plutus.Contracts
+    Plutus.Contracts.Auction
+    Plutus.Contracts.Crowdfunding
+    Plutus.Contracts.Currency
+    Plutus.Contracts.ErrorHandling
+    Plutus.Contracts.Escrow
+    Plutus.Contracts.Future
+    Plutus.Contracts.Game
+    Plutus.Contracts.GameStateMachine
+    Plutus.Contracts.GameStateMachine.Coverage
+    Plutus.Contracts.Governance
+    Plutus.Contracts.MultiSig
+    Plutus.Contracts.MultiSigStateMachine
+    Plutus.Contracts.PingPong
+    Plutus.Contracts.Prism
+    Plutus.Contracts.Prism.Credential
+    Plutus.Contracts.Prism.Mirror
+    Plutus.Contracts.Prism.StateMachine
+    Plutus.Contracts.Prism.STO
+    Plutus.Contracts.Prism.Unlock
+    Plutus.Contracts.PubKey
+    Plutus.Contracts.SealedBidAuction
+    Plutus.Contracts.SimpleEscrow
+    Plutus.Contracts.Stablecoin
+    Plutus.Contracts.Swap
+    Plutus.Contracts.TokenAccount
+    Plutus.Contracts.Tutorial.Escrow
+    Plutus.Contracts.Tutorial.EscrowStrict
+    Plutus.Contracts.Uniswap
+    Plutus.Contracts.Uniswap.OffChain
+    Plutus.Contracts.Uniswap.OnChain
+    Plutus.Contracts.Uniswap.Pool
+    Plutus.Contracts.Uniswap.Trace
+    Plutus.Contracts.Uniswap.Types
+    Plutus.Contracts.Vesting
+
+  hs-source-dirs:     src
+  default-language:   Haskell2010
+  default-extensions:
+    DeriveFoldable
+    DeriveFunctor
+    DeriveGeneric
+    DeriveLift
+    DeriveTraversable
+    ExplicitForAll
+    GeneralizedNewtypeDeriving
+    ImportQualifiedPost
+    MultiParamTypeClasses
+    ScopedTypeVariables
+    StandaloneDeriving
+
+  ghc-options:
+    -Wall -Wnoncanonical-monad-instances -Wunused-packages
+    -Wincomplete-uni-patterns -Wincomplete-record-updates
+    -Wredundant-constraints -Widentities -fobject-code
+    -fno-ignore-interface-pragmas -fno-omit-interface-pragmas
+
+  -- See Plutus Tx readme
```

Comment:
Apply the same change as below.

Reply:
Applied to all relevant cabal files in last commit

Reply:
This one hasn't been fixed yet